static/sass/ds/6-components/_summary-box.scss

```diff
@@ -0,0 +1,24 @@
+.c-summary-box {
+  border-bottom: 2px solid $color-white-off;
+  padding-bottom: $size-xl;
+
+  @include mq($until: bp-l) {
+    background: $color-white-pure;
```

Comment:
You might get away with letting the `has-bg-white` class sit on the `c-summary-box` div and then you won't need to say this needs to be the color in your media query